Changes LGTM, but there is still a failing mypy case:

``` python-traceback
src/bokeh/core/serialization.py: note: In member "_decode_bytes" of class "Deserializer":
src/bokeh/core/serialization.py:618:16: error: Incompatible return value type
(got "bytes | memoryview[int]", expected "bytes")  [return-value]
            return buffer.data
                   ^~~~~~~~~~~
Found 1 error in 1 file (checked 661 source files)
```

--- a/src/bokeh/core/serialization.py
+++ b/src/bokeh/core/serialization.py
@@ -600,7 +600,7 @@ class Deserializer:
         entries = obj.get("entries", [])
         return { self._decode(key): self._decode(val) for key, val in entries }
 
-    def _decode_bytes(self, obj: BytesRep) -> bytes:
+    def _decode_bytes(self, obj: BytesRep) -> bytes | memoryview[int]:
         data = obj["data"]
 
         if isinstance(data, str):

